node/resolvers/items.ts

```diff
@@ -112,9 +112,15 @@ export const mutations = {
     const cleanItems = items.map(
       ({ options, index, uniqueId, ...rest }) => rest
     )
-    const withOptions = items.filter(
-      ({ options }) => !!options && options.length > 0
-    )
+
+    // Filter only the items with options and ensure that they have a proper index property
```

Comment:
same as Lucas commented below, but I also think that there is no necessity for the comment

Reply:
thanks for the response @wdsrocha , I've answered in the other thread, the comment was because I've prioritized performance over legibility, then I added the comment because the reduce was trick haha

Reply:
No worries, I agree that the comment + `reduce` is great, but given that we will go with `map` + `filter`, I'm afraid the comment can end up making more harm than good. Does that make sense to you?

```suggestion
```